vm/systemSmartContracts/delegation.go

```diff
@@ -991,6 +993,24 @@ func (d *delegation) changeOwner(args *vmcommon.ContractCallInput) vmcommon.Retu
 	d.eei.SetStorageForAddress(d.delegationMgrSCAddress, args.CallerAddr, []byte{})
 	d.eei.SetStorage([]byte(ownerKey), args.Arguments[0])
 
+	globalFund, err := d.getGlobalFundData()
```

Comment:
make a new function for these 2 logEntry creations.

Reply:
done